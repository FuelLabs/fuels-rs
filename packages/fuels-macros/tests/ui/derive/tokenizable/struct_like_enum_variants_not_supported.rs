use fuels_macros::Tokenizable;

#[derive(Tokenizable)]
enum SomeEnum {
    A,
    B { something: u64 },
}

fn main() {}
