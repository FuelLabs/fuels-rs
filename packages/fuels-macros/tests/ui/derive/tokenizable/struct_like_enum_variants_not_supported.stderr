error: struct like enum variants are not supported
 --> tests/ui/derive/tokenizable/struct_like_enum_variants_not_supported.rs:6:7
  |
6 |     B { something: u64 },
  |       ^^^^^^^^^^^^^^^^^^
